languages/php/src/BitwardenSDK.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Bitwarden\Sdk;
+
+use Bitwarden\Sdk\Schemas\AccessTokenLoginRequest;
+use Bitwarden\Sdk\schemas\ClientSettings;
+use Bitwarden\Sdk\Schemas\Command;
+use FFI;
+use Swaggest\JsonDiff\Exception;
+
+
+class BitwardenSDK
```

Comment:
Can we rename this to `BitwardenClient` to keep inline with the other bindings?

Reply:
renamed